script.sinaweibo/resources/language/resource.language.en_gb/strings.po

```diff
@@ -0,0 +1,178 @@
+# Kodi Media Center language file
+# Addon Name: Twitter Tweets
+# Addon id: script.twittertweets
+# Addon Provider: Lunatixz
```

Comment:
Please change author and addon name.... Could you also remove the two "twitter tweet" images (ie Icon, fanart).

Please change the your plugins Icon we do not allow transparent icons. THX